pkg/solana/chain.go

```diff
@@ -232,29 +232,60 @@ func newChain(id string, cfg *config.TOMLConfig, ks loop.Keystore, lggr logger.L
 	return &ch, nil
 }
 
-// ChainService interface
-func (c *chain) GetChainStatus(ctx context.Context) (relaytypes.ChainStatus, error) {
+func (c *chain) LatestHead(_ context.Context) (types.Head, error) {
+	sc, err := c.getClient()
+	if err != nil {
+		return types.Head{}, err
+	}
+
+	latestBlock, err := sc.GetLatestBlock()
```

Comment:
There is a way to specify the commitment level if we need. I'm not sure which level this defaults to and maybe we don't need to specify.

Reply:
it just uses the default commitment in the config: https://github.com/smartcontractkit/chainlink-solana/blob/5b8d287fd054c42d22c1362fbcdef9f3622f505d/pkg/solana/client/client.go#L257

which should be "confirmed"